### DISCUSSION TITLE
Meta data in HTML header

### DISCUSSION THREAD

**Discussion by ischoegl** [executablebooks/meta#734] [Q&A] [answered]
I am currently looking for ways to include meta data in the HTML header, e.g.
```
<meta name="robots" content="noindex">
```
in the `<head>` section of the output files. Is there an easy way to do this? (As far as I am aware of, this cannot be done using CSS.) I checked both [page metadata](https://jupyterbook.org/en/stable/content/metadata.html) and [advanced](https://jupyterbook.org/en/stable/advanced/html.html) documentation, but don't see that any of my attempts are affecting the output.

**Comment by chrisjsewell** [comment, marked as answer]
Heya, I think you are after: https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#setting-html-metadata

**Reply by ischoegl** [reply]
```
---
html_meta:
  "robots": "noindex"
[...]
---
```
indeed resolved this. Btw, this was quick - thank you!!

**Reply by chrisjsewell** [reply]
cheers!
